### DISCUSSION TITLE
rust-course/增加面经

### DISCUSSION THREAD

**Discussion by Kasper4649** [sunface/rust-course#366] [内容建议]
目前网上 Rust 面经还是太少，不知道面试会问哪些，考虑增加这方面内容吗

**Comment by sunface** [comment]
面试经验是一个总结性的而不是探索性的话题，所以，很尴尬，我也不知道会问哪些哈哈。。不过这个建议可以先保留在这里，以后如果有机会，可以回头写写～

**Comment by Kasper4649** [comment]
学习 Rust 过程中找到的几篇面经，汇总一下😄。

[飞书 Rust 实习](https://blog.kuangjux.top/2021/10/22/%E9%A3%9E%E4%B9%A6Rust%E5%AE%9E%E4%B9%A0%E9%9D%A2%E8%AF%95/)
[飞书 Rust](https://zhuanlan.zhihu.com/p/411979704)
[字节跳动 Rust/C++ 实习](https://www.nowcoder.com/discuss/201045)
[字节跳动 Rust/C++ 实习](https://www.nowcoder.com/discuss/538078)

**Reply by sunface** [reply]
哇。。感谢分享

**Reply by Kasper4649** [reply]
可以考虑在附录里加个面经分享，贴链接即可，让大家一起汇总学习。

**Reply by sunface** [reply]
考虑放在 "最佳实践" 章节中，你觉得如何

**Reply by Kasper4649** [reply]
都可以👌
